package test;

option cc_generic_services = true;

message EchoRequest {
    required string query = 1;
}

message EchoResponse {
    required string result = 1;
}

service EchoService {
    rpc Echo(EchoRequest) returns (EchoResponse);
}
